packages/synapse-react-client/src/lib/style/components/facet_nav/_facet-nav-panel.scss

```diff
@@ -91,15 +91,12 @@ $facetNavPanel-btn-margin: 0 1px;
     }
   }
 }
-.SaveFiltersButtonContainer {
+.SaveFiltersButton {
+  display: block;
   margin: auto;
-  .SaveFiltersButton {
-    display: block;
-    margin: auto;
-    width: 250px;
-    min-width: 150px;
-    min-height: 20px;
-    padding: 5px;
-    font-size: 16px;
-  }
+  width: 250px;
+  min-width: 150px;
+  min-height: 20px;
+  padding: 5px;
+  font-size: 16px;
```

Comment:
Honestly with the bottom divider now present, the centering looks kind of weird to me. I would consider just removing the custom styling on the button and just rendering a regular contained/primary color confirm button.

Reply:
Works for me! Updated in the next commit -- 
![feature_FacetNavPanel_Modal_primarybutton](https://user-images.githubusercontent.com/26949006/236921785-dd0bdfb2-c2d7-4737-8573-17215e3252e0.png)